.github/workflows/release.yaml

```diff
@@ -83,6 +83,7 @@ jobs:
           REGISTRY: ghcr.io
         run: |
           echo ${{ secrets.GITHUB_TOKEN }} | helm registry login -u twingate --password-stdin $REGISTRY
+          helm dependency build ./deploy/twingate-operator
```

Comment:
We added helm dependency and forgot to add this. 
Release would have failed if we tried